ERCS/erc-7811.md

```diff
@@ -0,0 +1,222 @@
+---
+eip: 7811
+title: Wallet Asset Discovery
+description: JSON-RPC method for wallets to share a user’s full asset list with Dapps, including assets not easily found through on-chain data alone
```

Comment:
```suggestion
description: JSON-RPC method for wallets to share the full asset list owned by a given account address with Dapps, including assets not easily found through on-chain data alone
```

Could you clarify Is it an user or a account (wallet) address? 

For example many traditional wallet app user have a seed phrase that enables them to have multiple account addresses by BIP32 plus on different chain.